links/src/utils/getParent.ts

```diff
@@ -0,0 +1,26 @@
+import { ToTree, Tree, GetSiblings } from "./generateLinks";
+
+// getParents(tree, '/cancerqld/research') get parent_id of slug
```

Comment:
Make sure your comments are verbose and relevant [How to comment using JSDoc notation](https://devhints.io/jsdoc)